docs/aws_storage.md

```diff
@@ -0,0 +1,41 @@
+# AWS Storage setup instruction for Loki
+
+## Example config via yaml file
+```yaml
+schema_config:
+  configs:
+  - from: 0
+    store: aws
+    object_store: aws
+    schema: v9
+    index:
+      prefix: dynamodb_table_name
+      period: 0
```

Comment:
You probably want to set this to `168h`, so the tables rotate every week.  But of course, table_manager is required then.

Reply:
That makes sense to me now. Still not super familiar with the codebase yet, sorry about that.

I do have few questions related to table-manager and index table rotating.

1. Is it table-manager's responsibility to maintain index tables? Right now, I believe there is an error in table-manager, that creates tables from epoch instead of from configuration. 

You can follow the conversation in CNCF slack's #cortex channel.
```
Alex Milushev [8:06 AM]
level=info ts=2019-01-28T16:03:56.073716287Z caller=table_manager.go:207 msg="synching tables" num_expected_tables=2561 expected_tables=2561
level=info ts=2019-01-28T16:03:56.105879254Z caller=table_manager.go:336 msg="creating table" table=cortex_0
level=error ts=2019-01-28T16:03:56.583468248Z caller=table_manager.go:195 msg="error syncing tables" err="AccessDeniedException: User: arn:aws:sts::331840785346:assumed-role/core-worker/i-09e22af86e7ddcf57 is not authorized to perform: dynamodb:CreateTable on resource: arn:aws:dynamodb:us-east-1:331840785346:table/cortex_0\n\tstatus code: 400, request id: EBDF68R79N100RV46LMDPUJJLBVV4KQNSO5AEMVJF66Q9ASUAAJG"
```
```
Bryan Boreham [8:07 AM]
2561 tables at one per week is ~49 years so I suspect it's trying to create every table since the epoch
Definitely needs a bug report
Maybe as work-around put the 'from' date well into the future?
BTW you don't need to run table-manager, especially if you aren't creating tables every week.
```

2. What is the roadmap/timeline for index table? 

I read from the reddit thread that boltdb will eventually replace dynamodb/big table.  If that is a near term feature, I probably won't invest in table-manager too much for my projects.

reddit reference: https://www.reddit.com/r/devops/comments/a754si/grafanas_new_log_analysis_product/ec1cxel/

Reply:
> 1. Is it table-manager's responsibility to maintain index tables? Right now, I believe there is an error in table-manager, that creates tables from epoch instead of from configuration.

Thats right.  You can configure this (there is a flag) - generally the `from` field is used.  Feel free to file a bug over at Cortex and we'll make it clearer.

> 2. What is the roadmap/timeline for index table?

DynamoDB support will always be around, as will Bigtable.  The Boltdb stuff is new and is complimentary - you can use it locally and eventually I want to have it flush the BoltDB indexes to S3.  But thats probably a few months off.